docs/gen_two_proposal/diagrams/seat_massage_sequence.puml

```diff
@@ -0,0 +1,192 @@
+    uri: http://[::1]:50010
+    context: "sample-cabin"
+  }
+]
+end note
+
+
+"Massage Controller" -> "Digital Twin Provider - Cabin": Get on endpoint for "sample-cabin" - request
+
+"Massage Controller" <- "Digital Twin Provider - Cabin": Get - response
+note left
+Get response's payload (in JSON):
+
+ {
+  "@dtid": "sample-cabin",
+  "HVAC": {
+    "@dtid": "sample-hvac",
+    "@metadata": {
+      "model": "dtmi:sdv.HVAC;1"
+    }
+  },
+  "Infotainment": {
+    "@dtid": "sample-infotainment",
+    "@metadata": {
+      "model": "dtmi:sdv.Infotainment;1"
+  },
+  "FrontLeftSeat": {
+    "@dtid": "sample-front-left-seat",
+    "@metadata": {
+      "model": "dtmi:sdv.FrontSeat;1"
+  },
+  "FrontRightSeat": {
+    "@dtid": "sample-front-right-seat",
+    "@metadata": {
+      "model": "dtmi:sdv.FrontSeat;1"
+  },
+  "@metadata": {
+    "model": "dtmi:sdv.Cabin;1"
+  }
+} 
+end note
+
+"Massage Controller" -> "In-Vehicle Digital Twin Service": FindByDigitalTwin("sample-front-right-seat") - request
+
+"Massage Controller" <- "In-Vehicle Digital Twin Service": FindByDigitalTwin - response 
+note left
+FindByDigitalTwin response:
+
+endpoints: [
+  {
+    dtid: "sample-front-right-seat"
+    model: "dtmi:sdv:FrontSeat;1"
+    protocol: "grpc"
+    operations: [ "Get", "Set" ]
+    uri: http://[::1]:60010
+    context: "sample-front-right-seat"
+  }
+]
+end note
+
+"Massage Controller" -> "Digital Twin Provider - FrontSeat": Get on endpoint for "sample-front-seat" - request
+
+"Massage Controller" <- "Digital Twin Provider - FrontSeat": Get - response
+note left
+Get response's payload (in JSON):
+
+{
+  "@dtid": "sample-front-seat",
+  "SeatMassager": {
+    "@dtid": "sample-front-right-seat-massager",
+    "@metadata": {
+      "model": "dtmi:sdv.PremiumAirbagSeatMassager;1"
+    }
+  }
+  "@metadata": {
+    "model": "dtmi:sdv.FrontSeat;1"
+  }
+}
+end note
+
+"Massage Controller" -> "In-Vehicle Digital Twin Service": FindByDigitalTwin("sample-front-right-seat-massager") - request
+
+"Massage Controller" <- "In-Vehicle Digital Twin Service": FindByDIgitalTwin - response 
+note left
+FindByDigitalTwin response:
+
+endpoints: [
+  {
+    dtid: "sample-front-right-seat-massager"
+    model: "dtmi:sdv:PremiumAirbagSeatMassager;1"
+    protocol: "grpc"
+    operations: [ "Get", "Set" ]
+    uri: http://[::1]:70010
+    context: "sample-front-right-seat-massager"
+  }
+]
+end note
+
+loop until stopped
+  "Massage Controller" -> "Digital Twin Provider - PremiumAirbagSeatMassager": Set on "sample-front-right-seat-massager" - request
+  note right
+
+  An example of one of the set calls:
+
+  {
+    "@dtid": "sample-front-right-seat-massager",
+    "MassageAirBags": {
+      "0":  "25",
+      "1":  "25",
+      "2":  "50",
+      "3":  "50",
+      "4":  "75",
+      "5":  "75",
+      "6":  "100",
+      "7":  "100",
+      "8":  "75",
+      "9":  "75",
+    },
+    "@metadata": {
+      "model": "dtmi:sdv.PremiumAirbagSeatMassager;1"
```

Comment:
I know this is an example of a set call for this specific seat massage scenario, but I don't think the "model" field needs to be included since I already know which Digital Twin instance I'm targeting and the property.

If the metadata field is required, the field can just be empty "@metadata: {}".

Reply:
The model id here is for the property, not the enclosing digital twin.